pyvrp/repair/tests/test_greedy_repair.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+from numpy.testing import assert_, assert_equal, assert_raises
+
+from pyvrp import CostEvaluator, DynamicBitset, RandomNumberGenerator, Solution
+from pyvrp.repair import greedy_repair
+from pyvrp.tests.helpers import read
+
+
```

Comment:
Could we also add a test here that inserts a client into an empty route?

Reply:
It's an absolute pain to document these overloads. Let's not do this again.